src/Components/EntityHeaders/EntityHeaderSubmission.tsx

```diff
@@ -0,0 +1,85 @@
+import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
+import { Avatar, BoxProps, Flex, Text } from "@artsy/palette"
+import { EntityHeaderSubmission_submission$data } from "__generated__/EntityHeaderSubmission_submission.graphql"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface EntityHeaderSubmissionProps extends BoxProps {
+  submission: EntityHeaderSubmission_submission$data
+  displayAvatar?: boolean
+  displayLink?: boolean
+  FollowButton?: JSX.Element
+}
+
+const EntityHeaderSubmission: FC<EntityHeaderSubmissionProps> = ({
```

Comment:
Was it not possible to use the existing `EntityHeader` ?

Reply:
Not really, `EntityHeader`[ is deprecated](https://github.com/artsy/palette/blob/82a2e2171901d1519dc9aded0032d68ad9630ece/packages/palette/src/elements/EntityHeader/EntityHeader.tsx#L23) and I needed a new submission header that displays the artist name and the submission title.